pkg/storages/azure/folder.go

```diff
@@ -99,40 +100,40 @@ func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder,
 		tryTimeout = defaultTryTimeout
 	}
 
-	pipeLine := azblob.NewPipeline(
-		credential,
-		azblob.PipelineOptions{
-			Retry:      azblob.RetryOptions{TryTimeout: time.Duration(tryTimeout) * time.Minute},
```

Comment:
Need to figure out where try timeout goes now, also remove more code around here calculating the now unused `tryTimeout` variable